protoc-gen-swagger/genswagger/template.go

```diff
@@ -567,11 +574,15 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 					} else {
 						lastField := b.Body.FieldPath[len(b.Body.FieldPath)-1]
 						schema = schemaOfField(lastField.Target, reg, customRefs)
+						if schema.Description != "" {
+							desc = schema.Description
+						} else {
+							desc = fieldProtoComments(reg, lastField.Target.Message, lastField.Target)
+						}
 					}
 
-					desc := ""
 					if meth.GetClientStreaming() {
-						desc = "(streaming inputs)"
+						desc = desc + "(streaming inputs)"
```

Comment:
This can just be `desc += " (streaming inputs)"` right? (Also notice the space before the first bracket)

Reply:
Thanks! Fixed.